ex02.ex

```diff
@@ -53,7 +53,10 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
+  first2equal = fn [a, a | _ ] -> true
```

Comment:
## -1

This is almost right. But why does `first2equal([1])` return `false` an `first2equal([])` raise an exception?